app/room.py

```diff
@@ -41,32 +42,27 @@ def __init__(self, Room_name = "",Room_type = ""):
 
 	"""
 	Allocate a person to a room at random depending on if the person 
-	wants allocation{ also yes for staff }. 
+	wants allocation { also yes for staff }. 
 	check if room is full.
-		NB
-			fellow are allocated either office or living space
-			staff are allocated only office.
 	"""	
-	def allocate_member_a_room(self,name = "",id_no = 0):
+	def allocate_member_a_room(self,name = "",id_no = 0, room_type = ""):
 		self.name = name
 		self.id_no = id_no
+		self.list = {"FELLOW":"LIVINGSPACE","STAFF":"OFFICE"}
+		self.room_type = room_type
 
 		#1. Add to the list of current occupents in the room instance
```

Comment:
Follow pep8 commenting. Single line comments have a space after `#` sign . Checkout [this](https://www.python.org/dev/peps/pep-0008/#id31)